test/unit/util_color.js

```diff
@@ -11,25 +11,25 @@ test('decimalToHex', function (t) {
 });
 
 test('decimalToRgb', function (t) {
-    t.deepEqual(color.decimalToRgb(0), {r:0,g:0,b:0});
-    t.deepEqual(color.decimalToRgb(1), {r:0,g:0,b:1});
-    t.deepEqual(color.decimalToRgb(16777215), {r:255,g:255,b:255});
-    t.deepEqual(color.decimalToRgb(-16777215), {r:0,g:0,b:1});
-    t.deepEqual(color.decimalToRgb(99999999), {r:245,g:224,b:255});
+    t.deepEqual(color.decimalToRgb(0), {r: 0,g: 0,b: 0});
```

Comment:
If we are going to enforce a space after the `:` we should probably also enforce a space after the `,`.